.github/workflows/license-check.yml

```diff
@@ -1,8 +1,9 @@
 name: License Template Check
 
 on:
-  pull_request:
-    types: [synchronize, opened, reopened, ready_for_review]
+  workflow_dispatch:
+#  pull_request:
+#    types: [synchronize, opened, reopened, ready_for_review]
```

Comment:
Did you mean to leave this commented out instead of deleting it or retaining it?